Assets/Mirror/Core/NetworkManager.cs

```diff
@@ -127,6 +127,10 @@ public class NetworkManager : MonoBehaviour
         public static List<Transform> startPositions = new List<Transform>();
         public static int startPositionIndex;
 
+        [Header("Security")]
+        [Tooltip("For security, it can be a good idea to disconnect a player if they were able to trigger a runtime Exception.\nThis could be prevent components being accessed in an undefined state, which may be an attack vector for exploits.\nHowever, some games may want to allow exceptions in order to not interrupt the player's experience.")]
+        public bool exceptionsDisconnect = false;
```

Comment:
I'd be very much on favor of having this on by default.

Reply:
true